server/server.go

```diff
@@ -888,13 +892,15 @@ func (s *Server) Start() error {
 		s.ProjectCmdOutputHandler.Handle()
 	}()
 
-	server := &http.Server{Addr: fmt.Sprintf(":%d", s.Port), Handler: n}
+	tlsConfig := &tls.Config{GetCertificate: s.GetSSLCertificate}
+
+	server := &http.Server{Addr: fmt.Sprintf(":%d", s.Port), Handler: n, TLSConfig: tlsConfig}
 	go func() {
 		s.Logger.Info("Atlantis started - listening on port %v", s.Port)
 
 		var err error
 		if s.SSLCertFile != "" && s.SSLKeyFile != "" {
-			err = server.ListenAndServeTLS(s.SSLCertFile, s.SSLKeyFile)
+			err = server.ListenAndServeTLS("", "")
```

Comment:
If the arguments are always empty when they are passed in, then can we remove the arguments from the function declaration?

```suggestion
			err = server.ListenAndServeTLS()
```

Reply:
I get:
`server/server.go:903:34: not enough arguments in call to server.ListenAndServeTLS
	have ()
	want (string, string)`
when I try this.

Reply:
ah interesting so we have to give it something. Too bad.